test/integration-tests/shared/protocolHelpers/seaport.ts

```diff
@@ -13,6 +14,9 @@ export const seaportV2Orders = JSON.parse(
   fs.readFileSync('test/integration-tests/shared/orders/SeaportV2.json', { encoding: 'utf8' })
 )
 export const seaportInterface = new ethers.utils.Interface(SEAPORT_ABI)
+export const seaportV2Interface = new ethers.utils.Interface(SEAPORT_V2_ABI)
+// @dev 0 addr for an order that was not sent through the OpenSea conduit
```

Comment:
> 0 addr
This isnt an address - its a bytes32 right?

Reply:
You're right! will change